### DISCUSSION TITLE
Lab Assignment #3

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/22S-ASIAAM-191A#21] [Assignments]
## Due 4/20 before class 

Please use this thread to post your [lab assignment's #3](https://albertkun.github.io/22S-ASIAAM-191A/assignments/week3/lab_assignment/) link, which is the GitHub pages URL. [Click here for a reminder on how to get it](https://albertkun.github.io/22S-ASIAAM-191A/labs/week1/3/#using-github-pages).

### Note 
If you are submitting in two parts, put the part in the link you are submitting

#### For example:
Part1 - https://albertkun.github.io/22s-asiaam-191a-sample/week3a/index.html
Part2 - https://albertkun.github.io/22s-asiaam-191a-sample/week3b/index.html

If you only have one page, then you can just submit the `index.html` link as per usual.

**Comment by shubhgupta2018** [comment]
Part 1: https://shubhgupta2018.github.io/22s-191a-assignments/week-3/part1.html
Part 2: https://shubhgupta2018.github.io/22s-191a-assignments/week-3/index.html

**Comment by ananyakodali17** [comment]
3A: https://ananyakodali17.github.io/asian-am-191a-labs/week3/part1.html
3B: https://ananyakodali17.github.io/asian-am-191a-labs/week3/index.html

**Comment by daikis12** [comment]
https://daikis12.github.io/22s-191a-assignments/week3/index.html

**Comment by ntran27** [comment]
https://ntran27.github.io/22s-asiaam-191a-labs/week3/part1.html

**Comment by emmashanson** [comment]
Part 1:https://emmashanson.github.io/22s-asianam-191a-labs/week%203/part1.html
Part 2: https://emmashanson.github.io/22s-asianam-191a-labs/week3b/index.html

**Comment by bittertang** [comment]
https://bittertang.github.io/22s-asiaam-191a-labs/week%203/part1.html

**Comment by XiongHer** [comment]
https://xiongher.github.io/22s-asiaam-191a-labs/week3/part1.html

**Comment by kris10fu** [comment]
https://kris10fu.github.io/22s-asiaam-191a-labs/week3/index.html

**Comment by stefuntran** [comment]
https://stefuntran.github.io/22s-asiaam-191a-labs/week%203a/index.html

**Comment by joonyloony** [comment]
https://joonyloony.github.io/22s-191a-assignments/week3/part1.html

https://joonyloony.github.io/22s-191a-assignments/week3/index.html (i tried but failed)

**Comment by willa-21** [comment]
https://willakurland.github.io/aa191-labs/Week-3/part1.html
https://willakurland.github.io/aa191-labs/Week-3/part2.html

**Comment by ashley-lu** [comment]
https://ashley-lu.github.io/s22-asiaam-191a-assignments/week3lab/index.html (spent too much time on this)

**Comment by kericsp** [comment]
Part1 - https://kericsp.github.io/aa191-labs/week3/week3a/index.html
Part2 - https://kericsp.github.io/aa191-labs/week3/week3b/index.html

**Comment by hajovel** [comment]
part 1: https://hajovel.github.io/211S-191A-Assignments/week3/map.html
part 2: https://hajovel.github.io/211S-191A-Assignments/week3/map2.html

**Comment by penelopelamb** [comment]
part 1: https://penelopelamb.github.io/22s-aas-191a-labs/week3/iceCream.html
part 2: https://penelopelamb.github.io/22s-aas-191a-labs/week3/index.html

**Comment by angelineeow** [comment]
https://angelineeow.github.io/22s-asiaam-191a-labs/week3/more_about_me.html

**Comment by codingdianat** [comment]
https://codingdianat.github.io/aa191-assignments/week1/

**Comment by spolcharoen** [comment]
https://spolcharoen.github.io/22s-asiaam-191a-labs/week3/part1.html

**Comment by NishuKrish** [comment]
Part 1: https://nishukrish.github.io/AsianAm191A-Nish/week3/index.html
Part 2: https://nishukrish.github.io/AsianAm191A-Nish/week3/myplaces.html

**Comment by kaittli** [comment]
Part1: https://kaittli.github.io/22s-asiaam-191a-labs/week3/part1.html
Part2: https://kaittli.github.io/22s-asiaam-191a-labs/week3/index.html

**Comment by antonyblin** [comment]
Part 1: https://antonyblin.github.io/aas191a-assignments/week3/food.html
Part 2: https://antonyblin.github.io/aas191a-assignments/week2/food.html

**Comment by lindseymardona** [comment]
https://lindseymardona.github.io/22s-asiaam-191a-labs/week3/index.html
